api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/MediaTranslatorImpl.java

```diff
@@ -0,0 +1,43 @@
+package org.openmrs.module.fhir2.api.translators.impl;
+
+import lombok.AccessLevel;
+import lombok.Setter;
+import org.hl7.fhir.r4.model.Media;
+import org.openmrs.Obs;
+import org.openmrs.module.fhir2.api.translators.MediaTranslator;
+import org.openmrs.module.fhir2.api.translators.ObservationStatusTranslator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.Nonnull;
+
+import static org.apache.commons.lang.Validate.notNull;
+
+@Component
+@Setter(AccessLevel.PACKAGE)
+public class MediaTranslatorImpl implements MediaTranslator {
```

Comment:
For the mappings to be added here, look at the ticket and the way we generally map those properties. Most of them have overlap with Obs, so the `ObservationTranslatorImpl` class would be a good place to start.